serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -79,7 +138,23 @@ Authentication for OpenAPI services in Serverless Workflow
 [.card-title]
 xref:testing-and-troubleshooting/basic-integration-tests-with-restassured.adoc[Testing your Serverless Workflow application using REST Assured]
 [.card-description]
-Testing your Serverless Workflow application using REST Assured
+Learn how to add unit tests in your workflow application using RestAssured
+--
+
+[.card]
+--
+[.card-title]
+xref:testing-and-troubleshooting/integration-tests-with-postgresql.adoc[Integration Test with PostgreSQL]
+[.card-description]
+Learn how to do integration tests on workflow applications that use PostgreSQL as persistence storage
+--
+
+[.card]
+--
+[.card-title]
+xref:testing-and-troubleshooting/mocking-opnapi-services-with-wiremock.adoc[Mocking OpenAPI services with WireMock]
+[.card-description]
+Learn how to mock external REST requests when testing your workflow applications
 --
 
```

Comment:
@ricardozanini  I miss this guide linking: xref:testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc[Mocking HTTP CloudEvents Sink with Wiremock] even the related PR is mentioned as included by this one. Should I add it in the guide PR or will be added by this one?

Reply:
I thought that you already included it in your PR, that's why I haven't added it here.

Reply:
@ricardozanini I just want to sync my PR with this one, Thanks!